pages/linux/gopass.md

```diff
@@ -0,0 +1,35 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+
+- Startup config:
+
+`gopass init`
+
+- Show all "keystores":
+
+`gopass mounts`
+
+- Mount a shared git "keystore":
+
+`gopass mounts add keystore_name location_git_repo`
+
+- Search interactively using a keyword:
+
+`gopass show keyword`
+
+- Search using a keyword:
+
+`gopass find keyword`
+
+- Sync all mounted "keystores":
+
+`gopass sync`
+
+- Show particular password entry:
+
+`gopass keystore_name/directory01/dir02/email@email.com`
```

Comment:
Can the user change any of these arguments? Perhaps we could wrap them in tokens as I've done in some of the above examples.

Also, could we use `path/to/directory` please instead of `directory01/dir02`? This'll make it more consistent with other pre-existing pages.

Reply:
:+1: